### PR TITLE
Emojicode bubble sort implementation.

### PR DESCRIPTION
**Pull request by julianschacher**
Emojicode documentation and more: https://www.emojicode.org/

### CODE REVIEW

**Comment by Butt4cak3** [comment]
> If it's emojicode, I will personally close the PR.

\- Me during today's stream

I must admit: It's hard to resist the urge.

**Comment by thbwd** [comment]
First off, I think it’s great someone finally adds a real language to this project. 😉

You can, however, improve your code by taking advantage of that fact that lists are in fact iterable:

```
🔂 number numbers 🍇
   😀 🔡 numbers 10❗️❗️
🍉
```

is enough to print the list.

**Comment by leios** [comment]
Feel free to review the code, if you want!

File: contents/bubble_sort/code/emojicode/bubble_sort.emojic [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      🍉
+    🍉
+  🍉
+🍉
+
+🏁 🍇
+  🍨 1.7 -3.0 2.5 2.0 -6.0 4.4 50.0 7.0 1.5 -4.3 0.3 🍆  ➡️ numbers
+
+  😀 🔤unordered:🔤❗️
+  🔂 i 🆕⏩⏩ 0 🐔 numbers❗️❗️ 🍇
```
Comment (thbwd) [inline contents/bubble_sort/code/emojicode/bubble_sort.emojic]: I’d recommend to change this to

```
🔂 number numbers 🍇
   😀 🔡 numbers 10❗️❗️
🍉
```

File: contents/bubble_sort/code/emojicode/bubble_sort.emojic [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    😀 🔡 🐽 numbers i❗️ 10❗️❗️
+  🍉
+
+  🛁🐇🥇 numbers❗️
+
+  😀 🔤ordered:🔤❗️
+  🔂 i 🆕⏩⏩ 0 🐔 numbers❗️❗️ 🍇
```
Comment (thbwd) [inline contents/bubble_sort/code/emojicode/bubble_sort.emojic]: Same here

**Comment by julianschacher** [comment]
@thbwd Thanks for the review! I applied the suggested changes.

**Comment by leios** [comment]
If @thbwd approves, I also approve. Thanks for the review!

**Pull request merged**